pl_bolts/losses/object_detection.py

```diff
@@ -0,0 +1,40 @@
+"""
+Generalized Intersection over Union (GIoU) loss (Rezatofighi et. al)
+"""
+
+import torch
+
+
+def giou_loss(pred: torch.Tensor, target: torch.Tensor) -> torch.Tensor:
```

Comment:
can we define separately `giou` as functional metric and here use as "supplement"?
so we gain more from this addition :]